CMakeLists.txt

```diff
@@ -2,6 +2,11 @@
 
 cmake_minimum_required(VERSION 3.9.4)
 cmake_policy(SET CMP0091 NEW)
+option(WASMEDGE_BUILD_ON_OHOS "Cross compiler WasmEdge on OpenHarmony." OFF)
+if(WASMEDGE_BUILD_ON_OHOS)
+  message("get build")
+  include(utils/ohos/CMakeLists.txt)
+endif()
```

Comment:
It's not possible to build OHOS with this cmakelists only.
So you should use your own cmakelists to include the origin wasmedge one.

Reply:
> It's not possible to build OHOS with this cmakelists only. So you should use your own cmakelists to include the origin wasmedge one.

I try to include the origin one in my cmakelists, but it can't transfer my definitions to the origin one.The `include` command just imports the definitions in included file but the included file can't  inherit my definitions which set in my cmakelists.

Reply:
Oh. Okay. Then, just copy to your own one. Because this can only be used by the script.